javascript-if/main.js

```diff
@@ -0,0 +1,78 @@
+/* exported isUnderFive,
+            isEven,
+            startsWithJ,
+            isOldEnoughToDrink,
+            isOldEnoughToDrive,
+            isOldEnoughToDrinkAndDrive,
+            categorizeAcidity,
+            introduceWarnerBro
+ */
+function isUnderFive(number) {
+  if (number < 5) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function isEven(number) {
+  if (number % 2 === 0) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function startsWithJ(string) {
+  if (string[0] === 'J') {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function isOldEnoughToDrink(person) {
+  if (person.age >= 21) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function isOldEnoughToDrive(person) {
+  if (person.age >= 16) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function isOldEnoughToDrinkAndDrive(person) {
+  if (person.age >= 21 && person.age >= 16) {
+    return false;
+  } else {
+    return false;
+  }
+}
+
+function categorizeAcidity(pH) {
+  if (pH === 7) {
+    return 'neutral';
+  } else if (pH < 7 && pH >= 0) {
+    return 'acidic';
+  } else if (pH > 7 && pH <= 14) {
+    return 'basic';
+  } else if (pH <= 0 || pH >= 14) {
+    return 'invalid pH level';
+  }
+}
+
+function introduceWarnerBro(name) {
+  if ((name === 'yakko') || (name === 'wakko')) {
+    return 'Come join the Warner Brothers!';
+  } else if (name === 'dot') {
+    return 'And the Warner sister Dot!';
+  } else {
+    return 'Goodnight Everybody!';
+  }
+}
```

Comment:
These messages don't match the spec:

---

![image](https://user-images.githubusercontent.com/16143771/177666460-645f4ba4-29bc-415a-9251-4505f74a4ea5.png)